### PR TITLE
chore(gateapi): Update gate api for 1.16.1

### PR DESCRIPTION
**Pull request by louisjimenez**
Closes #4927. This is 99% generated code from Swagger. This is a fairly large diff since the last time the gate client api was updated was for Spinnaker 1.13 and since than the version of Swagger used by kork has also changed.

I tested out these changes by running the test suite and doing some smoke testing.

### CODE REVIEW

File: cmd/canary/canary-config/save.go [hunk 1]
```diff
@@ -78,7 +78,7 @@ func saveCanaryConfig(cmd *cobra.Command, options SaveOptions) error {
 	var saveErr error
 	if resp.StatusCode == http.StatusOK {
 		_, saveResp, saveErr = gateClient.V2CanaryConfigControllerApi.UpdateCanaryConfigUsingPUT(
-			gateClient.Context, templateId, templateJson, map[string]interface{}{})
+			gateClient.Context, templateJson, templateId, map[string]interface{}{})
```
Comment (louisjimenez) [inline cmd/canary/canary-config/save.go]: This was the one non automated change. The order of the arguments to the api helper function had to be swapped.

Comment (plumpy) [reply cmd/canary/canary-config/save.go]: Just curious, but why was that? Did the order change in the gate API or something?

Comment (louisjimenez) [reply cmd/canary/canary-config/save.go]: Created an issue to track this, https://github.com/spinnaker/spinnaker/issues/4931

**Comment by student-06** [comment]
Hi, @louisjimenez @plumpy 
`spin pipeline execute --name execute-check --application test4 --output=jsonpath='{.id}'`
This is not working in `1.12` version but used to work in `1.7.2`. Is there anything I missing or is this an issue ?

**Comment by louisjimenez** [comment]
@student-06 is the pipeline executing? The output of this command did change in 1.12, https://github.com/spinnaker/spin/pull/217.

**Pull request merged**